scss/config/_abstracts.scss

```diff
@@ -4,6 +4,15 @@
 /// # Herman Config: Private Helpers
 /// @group config-utils
 
+// Functions
+// ---------
+$functions: (
+  'darken': get-function('darken'),
+  'desaturate': get-function('desaturate'),
+  'rgba': get-function('rgba'),
+  'convert': get-function('convert-units')
```

Comment:
Oh, I guess this one isn't really required, since the newer version of accoutrement-scale gets the function internally?

Reply:
It's not required, but the shortcut is nice. Could just use `'convert': 'convert-units'` for that, though.